node-packages/commons/src/logs/local-logger.ts

```diff
@@ -0,0 +1,41 @@
+const { addColors, createLogger, format, transports } = require('winston');
+import { getConfigFromEnv } from '../util/config';
+import { levels, colors } from './';
+
+export interface LogFn {
+  (...args: any[]): void;
+}
+
+export interface Logger {
+  fatal: LogFn;
+  error: LogFn;
+  warn: LogFn;
+  info: LogFn;
+  verbose: LogFn;
+  debug: LogFn;
+  trace: LogFn;
+  log: LogFn;
+}
+
+addColors(colors);
+
+export const logger = createLogger({
+  exitOnError: false,
+  levels,
+  format: format.combine(
+    format.colorize(),
+    format.splat(),
+    format.timestamp({ format: 'YYYY-MM-DD HH:mm:ss' }),
+    format.printf(
+      ({ timestamp, level, message }) => `[${timestamp}] [${level}]: ${message}`
+    )
+  ),
+  transports: [
+    new transports.Console({
+      level: getConfigFromEnv('LOGGING_LEVEL', 'info'),
```

Comment:
Naming things is hard, is `LOGGING_LEVEL` the right play? I also thought about `LOCAL_LOGGING_LEVEL` since this only affects `console.log` right now, but we discussed the possibility of having different severity levels for stdout and opensearch in the future.

Reply:
Decided on `CONSOLE_LOGGING_LEVEL`